File: Challenges/Coding World Cup/2023/Day 31 - NZ vs PAK - Dreaded Runs.java
public class Solution {
    public static int findLen(int n, int x, int[] a) {
        // Write your code here.
        int count = 0;
        for (int i = 0; i < n; i++) {
            if (a[i] == x) {
                count++;
            }
        }
        return n - count;
    }
}

File: Challenges/Coding World Cup/2023/Day 31 - NZ vs PAK - Dreaded Runs.cpp
int findLen(int n, int x, vector<int> &a)
{
    // Write your code here.
    int count = 0;
    for (int i = 0; i < n; i++)
    {
        if (a[i] != x)
            count++;
    }

    return count;
}
